src/cli/generate-project/templated/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "tikui",
+  "version": "2.0.0",
+  "description": "Pattern Library tool",
+  "keywords": [
+    "ui",
+    "ui kit",
+    "design system",
+    "pattern library"
+  ],
+  "homepage": "https://github.com/tikui/tikui.git#readme",
+  "bugs": {
+    "url": "https://github.com/tikui/tikui/issues"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Anthony Rey"
+  },
+  "contributors": [
+    {
+      "name": "Hippolyte Durix"
+    },
+    {
+      "name": "Jérémie Tisserand"
+    },
+    {
+      "name": "Arnaud Freismuth"
+    }
+  ],
```

Comment:
not sure it's needed as it's totally dynamic :) you should only keep the original author = you